README.md

```diff
@@ -1,32 +1,504 @@
+
+  US Data Storage
+
+  ```
+  #import "swrve.h"
+  - (BOOL) application:(UIApplication *)application didFinishLaunchingWithOptions:(NSDictionary *)launchOptions {
+     [Swrve sharedInstanceWithAppID:<app_id> apiKey:@"<api_key>" launchOptions:launchOptions];
+  }
+  ```
+
+  EU Data Storage
+
+  ```
+  #import "swrve.h"
+  - (BOOL) application:(UIApplication *)application didFinishLaunchingWithOptions:(NSDictionary *)launchOptions {
+     SwrveConfig* config = [[SwrveConfig alloc] init];
+     config.selectedStack = SWRVE_STACK_EU;
+     [Swrve sharedInstanceWithAppID:<app_id> apiKey:@"<api_key>" config:config launchOptions:launchOptions];
+  }
+  ```
+
+  For more information, see [How Do I Configure the Swrve SDK for EU Data Storage?](http://docs.swrve.com/faqs/sdk-integration/configure-sdk-for-eu-data-storage/) If you have any questions or need assistance configuring the SDK for EU data storage, please contact support@swrve.com.
+
+3. Replace <app_id> and <api_key> with your Swrve app ID and Swrve API key.
+
+4. (Optional): If you are planning to use or ask for location tracking permission, add the following keys to your `<App>-Info.plist` if not already there (edit the Value message to whatever text you want):
+
+  * `NSLocationAlwaysUsageDescription`. Value: “Location is needed to offer all functionality in this app”.
+  * `NSLocationWhenInUseUsageDescription`. Value: “Location is needed to offer all functionality in this app”.
+
+In-app Messaging
+-
+Integrate the in-app messaging functionality so you can use Swrve to send personalized messages to your app users while they’re using your app. If you’d like to find out more about in-app messaging, see [Intro to In-App Messages](http://docs.swrve.com/user-documentation/in-app-messaging/intro-to-in-app-messages/).
+
+Before you can test the in-app message feature in your game, you need to create an In App Message campaign in the Swrve Dashboard.
+
+For iOS, if you want to display an in-app message as soon as your view is ready, you must trigger the display event when the view controller is in the `viewDidAppear` state and not in `viewDidLoad` state.
+
+### In-app Messaging Deeplinks ###
+
+When creating in-app messages in Swrve, you can configure message buttons to direct users to perform a custom action when clicked. For example, you might configure a button to direct the app user straight to your app store. To enable this feature, you must configure deeplinks by performing the actions outlined below. For more information about creating in-app messages in Swrve, see [Creating In-App Messages](http://docs.swrve.com/user-documentation/in-app-messaging/creating-in-app-messages/).
+
+Integrate deeplinks to direct users to a sale, website or other target when they click an in-app message. Before handling deeplinks in Swrve, you’ll need to register a [custom URL scheme with iOS](https://developer.apple.com/library/ios/featuredarticles/iPhoneURLScheme_Reference/Introduction/Introduction.html).
```

Comment:
This line is confusing, since you can actually implement custom deeplinks without registering a custom url scheme with ios.